WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingView.swift

```diff
@@ -0,0 +1,222 @@
+                }
+
+                // Remaining characters and final destination
+                VStack(alignment: .leading) {
+                    Text(viewModel.remainingCharactersLabel)
+                        .subheadlineStyle()
+                        .padding(.bottom, Layout.contentVerticalSpacing)
+
+                    Text(viewModel.finalDestinationLabel)
+                        .subheadlineStyle()
+                }
+                .padding([.leading, .trailing], Layout.contentSpacing)
+                .padding([.top, .bottom], Layout.contentVerticalSpacing)
+                .background(Color(.systemGray6))
+
+                Spacer()
+            }
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .background(Color(.systemGray6))
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationTitle(Localization.adDestination)
+            .toolbar {
+                ToolbarItem(placement: .cancellationAction) {
+                    Button(Localization.cancel) {
+                        dismiss()
+                    }
+                }
+            }
+        }
+    }
+    private func sectionHeading(title: String) -> some View {
+        Text(title.uppercased())
+            .subheadlineStyle()
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .padding(Layout.sectionHeadingPadding)
+    }
+
+    private func destinationItem(title: String,
+                                 subtitle: String,
+                                 type: DestinationType,
+                                 showBottomDivider: Bool = false) -> some View {
+        HStack(alignment: .center) {
+            if type == viewModel.selectedDestinationType {
+                Image(systemName: "checkmark")
+                    .padding(.leading, Layout.contentSpacing)
+                    .padding(.trailing, Layout.contentHorizontalSpacing)
+                    .foregroundColor(Color(uiColor: .accent))
+            } else {
+                Image(systemName: "checkmark")
+                    .padding(.leading, Layout.contentSpacing)
+                    .padding(.trailing, Layout.contentHorizontalSpacing)
+                    .hidden() // Small hack to make the icon space consistent while not showing the icon.
+            }
+
+            VStack(alignment: .leading) {
+                Text(title)
+                    .bodyStyle()
+                    .padding(.top, Layout.contentVerticalSpacing)
+                Text(subtitle)
+                    .subheadlineStyle()
+                    .multilineTextAlignment(.leading)
+                    .padding(.bottom, Layout.contentVerticalSpacing)
+
+                if showBottomDivider {
+                    Divider()
+                        .background(Color(.systemGray3))
+                }
+            }
+        }
+        .frame(maxWidth: .infinity, alignment: .leading)
+        .background(Color(.systemBackground))
```

Comment:
We should use `listForeground` for the color here.

Reply:
This will be a member of the List in the next PR so I'll tackle removing this there.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/11642/commits/a281f6017ddb817382ceb168f344023a77f60c91